Foundation/Host.swift

```diff
@@ -33,6 +33,18 @@ open class Host: NSObject {
     }
     
     static internal func currentHostName() -> String {
+#if os(Windows)
+      var dwLength: DWORD = 0
+      GetComputerNameExA(ComputerNameDnsHostname, nil, &dwLength)
+
+      let hostname: UnsafeMutablePointer<Int8>? =
+          UnsafeMutableBufferPointer<Int8>
+              .allocate(capacity: Int(dwLength + 1))
+              .baseAddress
```

Comment:
In scl-foundation source, long lines arent wrapped.

Reply:
I really wish that `swift-format` was setup.  I'll join the lines if thats the style here.